code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/service/ServiceDiscoveryService.java

```diff
@@ -39,5 +39,4 @@
 
     List<Integer> getServiceInstanceUsedOrderIds(Service service);
 
```

Comment:
This function hasn't been removed.  The one removed was: getActiveGlobalServices()